README.md

```diff
@@ -20,7 +20,16 @@ To provision services, the brokerpak currently requires GCP credentials. The bro
 
 A Makefile supports the full local development lifecycle for the brokerpak.
 
-- `make` will build the brokerpak
+Make targets will run with the *cfplatformeng/csb* docker image. Make sure your local image is up to date or run
+
+`docker pull cfplatformeng/csb` to update it
+
+Alternatively, a custom image can be specified by setting the `CSB` environment variable
```

Comment:
I don't think this is true for GCP. It's in the Azure brokerpak makefile, but not the one for this repo.